vhdbuilder/packer/install-dependencies.sh

```diff
@@ -274,6 +274,13 @@ for p in ${packages[*]}; do
         evaluatedURL=$(evalPackageDownloadURL ${PACKAGE_DOWNLOAD_URL})
         installOras "${downloadDir}" "${evaluatedURL}" "${version}"
         echo "  - oras version ${version}" >> ${VHD_LOGS_FILEPATH}
+      done
+      ;;
+    "azure-acr-credential-provider")
+      for version in ${PACKAGE_VERSIONS[@]}; do
+        evaluatedURL=$(evalPackageDownloadURL ${PACKAGE_DOWNLOAD_URL})
+        downloadCredentialProvider "${downloadDir}" "${evaluatedURL}" "${version}"
+        echo "  - azure-acr-credential-provider version ${version}" >> ${VHD_LOGS_FILEPATH}
```

Comment:
let's add some UTs to make sure we're resolving the correct version(s)

Reply:
linux-vhd-content-tests.sh actually verifies that the packages are installed properly - https://github.com/Azure/AgentBaker/blob/6d6960b123ea691bb50c2254aa94f9ddad549296/vhdbuilder/packer/test/linux-vhd-content-test.sh#L144

(source I spent a lot of time debugging why I was hitting that error message and it was because I was only downloading 1.29 instead of 1.30 because I was overwriting the version lol)

Reply:
hmm okay, I know we have those spec tests now which is what I was referring to

we can come back and add those later if we're already getting e2e coverage though